### PR TITLE
fixed typo :)

### PR DESCRIPTION
**Pull request by TennisBowling**

### CODE REVIEW

File: docs/_interface/mining.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Now we're ready to start mining. In a new terminal session, run ethminer and con
 
 `ethminer` communicates with geth on port 8545 (the default RPC port in geth). You can
 change this by giving the [`--rpcport` option](../rpc/server) to `geth`. Ethminer will find
-get on any port. You also need to set the port on `ethminer` with `-P
+geth on any port. You also need to set the port on `ethminer` with `-P
```
Comment (holiman) [inline docs/_interface/mining.md:56]: Thanks, but.. While you're fixing this, there are some other errors that should be fixed aswell. The `etherbase`, I think is called `--miner.etherbase`, and `--rpcport` is `--http --http.port`. I suspect this page is pretty bitrotted and needs some loving

**Comment by holiman** [comment]
Oh, btw, I think @renaynay is already on this: https://github.com/ethereum/go-ethereum/pull/22265

**Comment by renaynay** [comment]
Yes in regards to deprecated flags, I have a PR open: #22265

**Comment by TennisBowling** [comment]
okay, let me know if I should close the pr

**Comment by karalabe** [comment]
Fixed in another PR

**Pull request closed without merging**